demo/src/screens/componentScreens/WithScrollReachedScreen/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {Component} from 'react';
```

Comment:
Personally I think that the 3 example screen files makes it a little difficult to understand what's going on. 
I think it's a great feature, but still a pretty small one, as a user I'm getting lost in all of these files when I'm trying to figure out how the API works.

Let's simplify it. I don't think there's really a need for showing how it works both on FlatList and ScrollView, the principles are similar, showing only on ScrollView it's good enough.

Reply:
ok, do you want it to be in a single file or a screen + component?

Reply:
Yes, let's simplify it. It's important that the example screen is easy to understand

Reply:
Merged files and removed `FlatList`